modules/compute/src/integrationTest/java/org/apache/ignite/internal/compute/ItComputeTestEmbedded.java

```diff
@@ -193,16 +201,64 @@ void changeJobPriorityLocally() {
         assertThat(execution3.changePriorityAsync(2), willBe(true));
 
         // Run 1 and 3 task
-        WaitLatchJob.latch.countDown();
+        countDownLatch.countDown();
+
+        // Tasks 1 and 3 completed successfully
+        assertThat(execution1.resultAsync(), willCompleteSuccessfully());
+        assertThat(execution3.resultAsync(), willCompleteSuccessfully());
+        assertThat(execution1.resultAsync().isDone(), is(true));
+        assertThat(execution3.resultAsync().isDone(), is(true));
+
+        // Task 2 is not completed
+        assertThat(execution2.resultAsync().isDone(), is(false));
+    }
+
+    @Test
+    void executesJobLocallyWithOptions() {
+        IgniteImpl entryNode = node(0);
+
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+
+        // Start 1 task in executor with 1 thread
+        JobExecution<String> execution1 = entryNode.compute()
+                .executeAsync(Set.of(entryNode.node()), units(), WaitLatchJob.class.getName(), countDownLatch);
+        await().until(execution1::statusAsync, willBe(jobStatusWithState(JobState.EXECUTING)));
+
+        // Start one more long lasting task
+        JobExecution<String> execution2 = entryNode.compute().executeAsync(Set.of(entryNode.node()), units(), LongJob.class.getName());
```

Comment:
What is the reason to use LongJob here? You can just create one more latch and pass it to job's arguments. That would be better.